test/e2e/e2e.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+
+source ./test/helper/helper.sh
+
+# E2E tests parameters for the test pipeline
+
+# Testing the openshift task,
+@test "[e2e] openshift task" {
+    [ -n "${E2E_OPENSHIFT_PARAMS_SCRIPT}" ]
+    
+    run kubectl delete taskrun --all
+    assert_success
+
+    kubectl delete secret regcred || true
+    run kubectl create secret generic regcred \
+        --from-file=.dockerconfigjson=$HOME/.docker/config.json \
+        --type=kubernetes.io/dockerconfigjson
+    assert_success
+    run kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "regcred"}]}'
+    assert_success
```

Comment:
Are we adding this so that we don't face any permission denial issue while accessing the image from registry.redhat.io . Is it because we didn't find any oc image in registry.access.redhat.com ?

Reply:
Yes. Some images are not exposed through `registry.access.redhat.com` so, I went with having some credentials instead.